Add Hugo example

msweeneydev opened the pull request:
Added a very basic Hugo example, based off the fantastic guide by @unicodeveloper :100:.

The `README.md` is based off @timothyis's Rust example with the main difference being a live example and some additional resources.

I would appreciate some feedback on the example, I have based this on a 'bare minimum' approach so that someone could pick it up and run with it, without needing to do any additional setup.

unicodeveloper commented on hugo/config.toml:
```diff
@@ -0,0 +1,4 @@
+baseURL = "http://example.org/"
```
Please change this to `/` so that when deployed it actually points to the base URL instead of `http://example.org/`

msweeneydev replied:
Changed :+1:

unicodeveloper commented on hugo/readme.md:
```diff
@@ -0,0 +1,20 @@
+
+## See Live
+
+The Now deployed verion of this repository can be found [here](https://hugo-now-example.now.sh).
```
verion => version

msweeneydev replied:
Thank you!

unicodeveloper commented on hugo/now.json:
```diff
@@ -0,0 +1,11 @@
+      "config": { "distDir": "public" }
+    }
+  ]
+}
```
```suggestion
{
  "version": 2,
  "public: true,
  "name": "hugo-now-example",
  "builds": [
    {
      "src": "build.sh",
      "use": "@now/static-build",
      "config": { "distDir": "public" }
    }
  ]
}
```

unicodeveloper replied:
We want all our examples to be able to have URLs that the source can be viewed easily whenever any comes across the deployment URL

msweeneydev replied:
Changed :+1:

unicodeveloper commented on hugo/readme.md:
```diff
@@ -0,0 +1,20 @@
+# @now/hugo Example
+
+This directory is a brief example of a static [Hugo](https://gohugo.io/) function using the [@now/static-build Builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/).
```
I'm not sure function is the right word here. It could be:

`This directory is a brief example of deploying a static [Hugo](https://gohugo.io/) website using the [@now/static-build Builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/).`

unicodeveloper commented on hugo/readme.md:
```diff
@@ -0,0 +1,20 @@
+# @now/hugo Example
```
This can just be `# Hugo`

msweeneydev replied:
Changed :+1:

unicodeveloper commented on hugo/readme.md:
```diff
@@ -0,0 +1,20 @@
+# @now/hugo Example
+
```
It would be awesome if the Live Demo was someone here. Just slightly below the `Hugo` header.

Something like: `[Live Demo](<link-to-live-demo>)`

msweeneydev replied:
Changed :+1:

unicodeveloper commented on hugo/readme.md:
```diff
@@ -0,0 +1,20 @@
+# @now/hugo Example
+
+This directory is a brief example of a static [Hugo](https://gohugo.io/) function using the [@now/static-build Builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/).
```
This can be reworded as:

```
In this example, we'll be deploying a static [Hugo](https://gohugo.io/) website to Now using the [@now/static-build Builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/).

You can run the following command `now init hugo` to fetch the example to your local machine.
```

msweeneydev replied:
Changed :+1:

timothyis commented:
Would you mind adding this to the root `now.json`, as a CI, and to the root readme?

msweeneydev commented:
@timothyis I have added to both of those files, redeployed to Now and aliased using the same format as the other examples.

I've also changed from using git submodules as it can lead to issues and gone for a straight clone of the theme instead. This doesn't affect anyone planning to build from this example in any way 👍
